gen/build_deploy/bash.py

```diff
@@ -670,23 +668,8 @@ def fill_template(base_name, format_args):
 
 
 def make_dcos_launch():
-    # NOTE: this needs to be kept in sync with build_dcos_launch.sh
-    work_dir = py.path.local.mkdtemp()
-    # the tree will not be in the working dir if run from downstream
-    tree_src = './'
-    if os.path.exists('ext/upstream'):
-        tree_src = './ext/upstream'
-    src_info = {'kind': 'git_local', 'rel_path': tree_src}
-    this_tree = GitLocalSrcFetcher(src_info, None, os.getcwd())
-    this_tree.checkout_to(str(work_dir))
-    # put the launch spec into the root of the tree before running pyinstaller
-    work_dir.join('dcos-launch.spec').write(pkg_resources.resource_string('launch', 'dcos-launch.spec'))
-    with work_dir.as_cwd():
-        subprocess.check_call(['pyinstaller', '--log-level=DEBUG', 'dcos-launch.spec'])
-    subprocess.check_call(['mv', str(work_dir.join('dist').join('dcos-launch')), 'dcos-launch'])
-    work_dir.remove()
-
-    return "dcos-launch"
+    # Make this its own function for easy mocking
+    subprocess.check_call(['./build_dcos_launch.sh'])
```

Comment:
This is a terrible approach. Reworking to invoke pkgpanda artifacts without re-starting a build (even though the build would be cached and therefore trivial)